test cases/common/259 preprocess/meson.build

```diff
@@ -13,9 +13,19 @@ bar_x = custom_target(
   capture: true,
 )
 
+foo_h = custom_target(
+  input: 'foo.x',
+  output: 'foo.h',
+  command: ['cat', '@INPUT@'],
+  capture: true,
+)
```

Comment:
This will fail on Windows which lacks cat, right? What about using the `fs.copyfile` instead?
```meson
foo_h = import('filesystem').copyfile('foo.x', 'foo.h')
```
This is a custom target, but it's also guaranteed to work on all platforms Meson supports